src/Deltas.proto

```diff
@@ -60,7 +60,7 @@ package Catalyst.Protocol.Deltas;
  */
 message CandidateDeltaBroadcast {
```

Comment:
Considering what happened to TransactionBroadcast, shouldn't these ones go to the "Wire" file
?